protocol/str.go

```diff
@@ -19,7 +19,7 @@ func NewDirSTR(str *merkletree.SignedTreeRoot) *DirSTR {
 
 // Serialize overrides merkletree.SignedTreeRoot.Serialize
 func (str *DirSTR) Serialize() []byte {
-	return append(str.SerializeInternal(), str.Policies.Serialize()...)
+	return append(str.SignedTreeRoot.SerializeInternal(), str.Policies.Serialize()...)
```

Comment:
I was getting an error during testing on my machine. I changed how I was testing and the error no longer appears so I'll change it back.